src/SprykerEco/Zed/UnzerGui/Communication/Controller/EditMarketplaceUnzerCredentialsController.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\UnzerGui\Communication\Controller;
+
+use Spryker\Service\UtilText\Model\Url\Url;
+use SprykerEco\Zed\UnzerGui\UnzerGuiConfig;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * @method \SprykerEco\Zed\UnzerGui\Communication\UnzerGuiCommunicationFactory getFactory()
+ */
+class EditMarketplaceUnzerCredentialsController extends AbstractUnzerCredentialsController
+{
+    /**
+     * @var string
+     */
+    protected const REQUEST_ID_PARENT_UNZER_CREDENTIALS = 'parent-id-unzer-credentials';
+
+    /**
+     * @param \Symfony\Component\HttpFoundation\Request $request
+     *
+     * @return \Symfony\Component\HttpFoundation\RedirectResponse|array<string,mixed>
+     */
+    public function indexAction(Request $request)
+    {
+        $idUnzerCredentials = $this->castId($request->get(static::PARAM_ID_UNZER_CREDENTIALS));
+        $dataProvider = $this->getFactory()->createUnzerCredentialsFormDataProvider();
+
+        /** @var \Generated\Shared\Transfer\UnzerCredentialsTransfer $unzerCredentialsTransfer */
+        $unzerCredentialsTransfer = $dataProvider->getData($idUnzerCredentials);
+
+        if (!$unzerCredentialsTransfer->getIdUnzerCredentials()) {
+            $this->addErrorMessage(static::MESSAGE_UNZER_CREDENTIALS_NOT_FOUND);
+
+            return $this->redirectResponse(static::REDIRECT_URL_DEFAULT);
+        }
+
+        $form = $this->getFactory()
+            ->getUnzerCredentialsEditForm(
+                $dataProvider->getData($idUnzerCredentials),
+                $dataProvider->getOptions($idUnzerCredentials),
+            )
+            ->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            return $this->handleUnzerCredentialsForm($request, $form);
+        }
+
+        return $this->prepareViewResponse($form, $idUnzerCredentials);
+    }
+
+    /**
+     * @param \Symfony\Component\HttpFoundation\Request $request
+     *
+     * @return \Symfony\Component\HttpFoundation\JsonResponse
+     */
+    public function tableAction(Request $request): JsonResponse
+    {
+        $parentIdUnzerCredentials = $this->castId($request->get(static::PARAM_ID_UNZER_CREDENTIALS));
+
+        $table = $this->getFactory()
```

Comment:
```suggestion
        $merchantUnzerCredentialsTable = $this->getFactory()
```

please check similar issues with variable naming at controllers

Reply:
Fixed.